filters/bevel/types.d.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference types="pixi.js" />
+declare namespace PIXI.filters {
+    class BevelFilter extends PIXI.Filter<{}> {
+        constructor(rotation?:number, thickness?:number, lightColor?:number, lightAlpha?:number, shadowColor?:number, shadowAlpha?:number);
+        rotation:number;
+        thickness:number;
+        lightColor:number;
+        lightAlpha:number;
+        shadowColor:number;
+        shadowAlpha:number;
+    }
+}
```

Comment:
Please make sure to copy the BevelFilter to the bundle types as well.

Reply:
Please check types in bundle/types.d.ts
is everything right?

Reply:
Looks good!